src/Chapter4.hs

```diff
@@ -611,6 +638,10 @@ Implement the 'Monad' instance for our lists.
   maybe a few) to flatten lists of lists to a single list.
 -}
 
+instance Monad List where
+    (>>=) :: List a -> (a -> List b) -> List b
+    Empty >>= _ = Empty
+    (Cons x xs) >>= f = append (f x) (xs >>= f)
```

Comment:
Alternatively, you could implement it with the `fmap` and function that would flatten list 🙂